falcon/util/misc.py

```diff
@@ -75,7 +75,9 @@
 
 # PERF(kgriffs): Avoid superfluous namespace lookups
 strptime: Callable[[str, str], datetime.datetime] = datetime.datetime.strptime
-utcnow: Callable[[], datetime.datetime] = datetime.datetime.utcnow
+utcnow: Callable[[], datetime.datetime] = functools.partial(
```

Comment:
note that this is a potentially breaking change, since the datetime now has a timezone.

I think we are fine keeping your change though

Reply:
Ah, right. This `utcnow` is just a performance alias (not published in our docs), so I don't think this is a breaking change. But it is better to be safe than sorry, I'll simply create a new a alias, `_utcnow`, and plop a `DeprecationWarning` on the old one.

Reply:
Privatized both now. Do you think we need a deprecation newsfragment for these?